js/protoblocks.js

```diff
@@ -527,6 +632,9 @@ class ProtoBlock {
     }
     // E.g., distance . Distance block will calculate geometrical distance between two pointa
```

Comment:
Maybe this "e.g." belongs in the comment block below (L636)

Reply:
Thank you for the rectification walter. I'll correct it and get back to you.